azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -134,6 +134,16 @@ func resourceArmMsSqlDatabase() *schema.Resource {
 				ValidateFunc:     validation.IsRFC3339Time,
 			},
 
+			"recoverable_database_id": {
```

Comment:
How does this work in relation to `restorable_dropped_database_id` ? they seem to be the same and i don't see a test for this?

Reply:
the same above

Reply:
and then this 
```suggestion
			"recover_database_id": {
```
?